src/main/java/com/alysaa/geyserupdater/bungee/BungeeUpdater.java

```diff
@@ -48,7 +48,7 @@ public void onEnable() {
         // Player alert if a restart is required when they join
         getProxy().getPluginManager().registerListener(this, new BungeeJoinListener());
         // Check if downloaded Geyser file exists periodically
-        getProxy().getScheduler().schedule(this, CheckBuildFile::checkBungeeFile, 30, 720, TimeUnit.MINUTES);
+        getProxy().getScheduler().schedule(this, () -> CheckBuildFile.checkSpigotFile(true), 30, 720, TimeUnit.MINUTES);
```

Comment:
Think this is a typo
```suggestion
        getProxy().getScheduler().schedule(this, () -> CheckBuildFile.checkBungeeFile(true), 30, 720, TimeUnit.MINUTES);
```